src/Tickets/Site_Health/Subsections/Plugins/Plugin_Data_Subsection.php

```diff
@@ -229,15 +229,20 @@ protected function generate_subsection(): array {
 	 * @return string List of previous versions.
 	 */
 	private function get_previous_versions(): string {
-		return Arr::to_list(
-			array_filter(
-				(array) tribe_get_option(
-					'previous_event_tickets_versions',
-					[]
-				)
-			),
+		$previous_versions = array_filter(
+			(array) tribe_get_option(
+				'previous_event_tickets_versions',
+				[]
+			)
+		);
+
```

Comment:
Bail out here with '' if `$previous_versions` is empty. Then just return the list on the next. Saves running the `Arr::tolist` on empty values.

Reply:
```suggestion
		if ( empty( $previous_versions ) ) {
			return '';
		}

```